pages/dao/[symbol]/proposal/components/instructions/Raydium/AddLiquidityToPool.tsx

```diff
@@ -0,0 +1,261 @@
+        })
+        const { validationErrors } = await isFormValid(schema, form)
+        setFormErrors(validationErrors)
+      })
+    }
+  }, [form.baseAmountIn, form.slippage])
+
+  useEffect(() => {
+    isFormValid(schema, form).then(({ validationErrors }) => {
+      setFormErrors(validationErrors)
+    })
+  }, [form.quoteAmountIn])
+
+  useEffect(() => {
+    handleSetInstructions(
+      { governedAccount: form.governedAccount?.governance, getInstruction },
+      index
+    )
+  }, [form])
+
+  const schema = yup.object().shape({
+    governedAccount: yup
+      .object()
+      .nullable()
+      .required('Program governed account is required'),
+    liquidityPool: yup.string().required('Liquidity Pool is required'),
+    baseAmountIn: yup
+      .number()
+      .moreThan(0, 'Amount for Base token should be more than 0')
+      .required('Amount for Base token is required'),
+    quoteAmountIn: yup
+      .number()
+      .moreThan(0, 'Amount for Quote token should be more than 0')
+      .required('Amount for Quote token is required'),
+    fixedSide: yup
+      .string()
+      .equals(['base', 'quote'])
+      .required('Fixed Side is required'),
+  })
+
+  return (
+    <>
+      <GovernedAccountSelect
+        label="Governance"
+        governedAccounts={governedMultiTypeAccounts}
+        onChange={(value) => {
+          handleSetForm({ value, propertyName: 'governedAccount' })
+        }}
+        value={form.governedAccount}
+        error={formErrors['governedAccount']}
+        shouldBeGoverned={shouldBeGoverned}
+        governance={governance}
+      ></GovernedAccountSelect>
+
+      <Select
+        label="Raydium Liquidity Pool"
+        value={form.liquidityPool}
+        placeholder="Please select..."
+        onChange={(value) =>
+          handleSetForm({ value, propertyName: 'liquidityPool' })
+        }
+        error={formErrors['liquidityPool']}
+      >
+        {Object.keys(liquidityPoolKeysList).map((pool, i) => (
+          <Select.Option key={pool + i} value={pool}>
+            {pool}
+          </Select.Option>
+        ))}
+      </Select>
+      {form.liquidityPool ? (
+        <>
+          <Input
+            label="Base Token Amount to deposit"
+            value={form.baseAmountIn}
+            type="number"
+            min={0}
+            max={10 ** 12}
+            onChange={(evt) =>
+              handleSetForm({
+                value: evt.target.value,
+                propertyName: 'baseAmountIn',
+              })
+            }
+            error={formErrors['baseAmountIn']}
+          />
+
+          <Select
+            label="Slippage (%)"
+            value={form.slippage}
+            onChange={(value) =>
+              handleSetForm({ value, propertyName: 'slippage' })
+            }
+            error={formErrors['slippage']}
+          >
+            {[0.5, 1, 2].map((value, i) => (
+              <Select.Option key={value.toString() + i} value={value}>
```

Comment:
Same, please refrain from using indexes in `key`s.

`key` only needs to be unique among its siblings (single loop)